tests/test_models.py

```diff
@@ -0,0 +1,36 @@
+from app.model import *
+import sys
+sys.path.append("..")
+
+
+class TestClassModel(object):
+    def test_languages(self):
+        """basic test Languages Collection"""
+        lang = Languages(language="russian", code="RUS",
+                         type="foreign", default=False)
+        lang.save()
+        assert Languages.objects.count() > 0
+        assert isinstance(lang, Languages)
+
+    def test_sentences(self):
+        """basic test Sentences Collection"""
+        lang = Languages.objects(language="russian").first()
+        text = Sentences(text="spasibo", lang=lang)
+        text.save()
+        assert Sentences.objects.count() > 0
```

Comment:
test that it is equal to 1?

Reply:
Since I am not using a dedicated test database, cannot test if it equal to 1. Will refactor